spec/observables/defer-spec.ts

```diff
@@ -72,4 +72,16 @@ describe('Observable.defer', () => {
     expectObservable(e1, unsub).toBe(expected);
     expectSubscriptions(source.subscriptions).toBe(sourceSubs);
   });
+
+  type(() => {
```

Comment:
This might be the wrong time to point this out... but I have NO IDEA what these tests do. I find them almost unreadable.